data/src/main/java/com/teamagam/gimelgimel/data/images/ImageUtils.java

```diff
@@ -1,25 +1,37 @@
 package com.teamagam.gimelgimel.data.images;
 
+import android.content.Context;
 import android.graphics.Bitmap;
-import android.graphics.BitmapFactory;
 import android.net.Uri;
 
 import com.teamagam.gimelgimel.data.common.ExternalDirProvider;
 import com.teamagam.gimelgimel.data.config.Constants;
 
-import java.io.ByteArrayOutputStream;
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
+import javax.inject.Inject;
+
+import id.zelory.compressor.Compressor;
+
 public class ImageUtils {
 
     private static final String IMAGE_TEMP_PREFIX = "image_";
     private static final String IMAGE_TEMP_TYPE = ".png";
     private static final String TEMP_DIR = "images";
     private static final String DATE_FORMAT = "dd_HH_mm_ss";
 
-    public static Uri getTempImageUri(ExternalDirProvider externalDirProvider){
+    @Inject
+    Context mContext;
+
+    @Inject
+    public ImageUtils() {
+    }
+
+    public Uri getTempImageUri(ExternalDirProvider externalDirProvider) {
```

Comment:
This method can now get its ExternalDirProvider with injection and not as a method parameter